sympy/core/numbers.py

```diff
@@ -4547,13 +4551,6 @@ def sympify_mpq(x):
     _sympy_converter[type(gmpy.mpq(1, 2))] = sympify_mpq
 
 
-def sympify_mpmath_mpq(x):
-    p, q = x._mpq_
-    return Rational(p, q, 1)
-
-_sympy_converter[type(mpmath.rational.mpq(1, 2))] = sympify_mpmath_mpq
```

Comment:
Somehow this got removed from the PR defeating the original purpose of compatibility with mpmath.